src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Controllers/HelloWorldController.cs

```diff
@@ -1,22 +1,75 @@
+using Asp.Versioning;
+using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
 using GovUk.Education.ExploreEducationStatistics.Common.Model;
 using GovUk.Education.ExploreEducationStatistics.Common.Model.Data;
 using GovUk.Education.ExploreEducationStatistics.Public.Data.Model;
 using GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Database;
 using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
 
 namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Controllers;
 
+[ApiVersion(1.0)]
+[ApiVersion(2.0)]
 [ApiController]
-[Route("/HelloWorld")]
+[Route("api/v{version:apiVersion}/HelloWorld")]
 public class HelloWorldController(PublicDataDbContext publicDataDbContext) : ControllerBase
 {
-    [HttpGet]
-    public async Task<ActionResult<string>> Get()
+    /// <summary>
+    /// List v1
+    /// </summary>
+    /// <remarks>
+    /// Test the API can list with `GET` in v1.
```

Comment:
this would be for v2 too wouldn't it?

Reply:
Nope, it gets superceded in v2 by the `ListV2` method!